packages/@azure/servicebus/data-plane/test/topicFilters.spec.ts

```diff
@@ -55,15 +55,22 @@ async function beforeEachTest(): Promise<void> {
   }
 
   ns = Namespace.createFromConnectionString(process.env.SERVICEBUS_CONNECTION_STRING);
-  topicClient = getSenderClient(ns, ClientType.TopicFilterTestTopic) as TopicClient;
-  subscriptionClient = getReceiverClient(
+
+  let clients = await getSenderReceiverClients(
     ns,
+    ClientType.TopicFilterTestTopic,
     ClientType.TopicFilterTestSubscription
-  ) as SubscriptionClient;
-  defaultSubscriptionClient = getReceiverClient(
+  );
+  topicClient = clients.senderClient as TopicClient;
+  subscriptionClient = clients.receiverClient as SubscriptionClient;
+
+  clients = await getSenderReceiverClients(
     ns,
+    ClientType.TopicFilterTestTopic,
     ClientType.TopicFilterTestDefaultSubscription
-  ) as SubscriptionClient;
+  );
+  topicClient = clients.senderClient as TopicClient;
+  defaultSubscriptionClient = clients.receiverClient as SubscriptionClient;
```

Comment:
Will this work as expected?

The second time you call `getSenderReceiverClients`, wouldn't the topic and subscription created in the first call be deleted? Yes, you will end up creating the topic again in the second call, but the subscription in the first call will not exist anymore.